src/common/smartContract/issuerToSmartContract.js

```diff
@@ -0,0 +1,43 @@
+const contractInstance = require("./contractInstance");
+const documentStoreAbi = require("./abi/documentStore.json");
+const tokenRegistryAbi = require("./abi/tokenRegistry.json");
+const { TYPES } = require("./constants");
+
+const issuerToSmartContract = (issuer, network) => {
```

Comment:
a pattern you might find fun, https://repl.it/repls/SeparateConventionalSystemintegrator
(not saying you have to apply it here)